atomics/T1518.001/T1518.001.yaml

```diff
@@ -66,15 +73,18 @@ atomic_tests:
 - name: Security Software Discovery - Sysmon Service
   auto_generated_guid: fe613cf3-8009-4446-9a0f-bc78a15b66c9
   description: |
-    Discovery of an installed Sysinternals Sysmon service using driver altitude (even if the name is changed).
+    Discovery of an installed Sysinternals Sysmon service using driver altitude (even if the name is changed) and various other methods.
 
-    when sucessfully executed, the test is going to display sysmon driver instance if it is installed.
+    when sucessfully executed, the test is going to display sysmon driver instance,processes,service if it is installed.
   supported_platforms:
   - windows
   executor:
     command: |
       fltmc.exe | findstr.exe 385201
-    name: command_prompt
+      Get-Process | Where-Object { $_.ProcessName -eq "Sysmon" }
+      Get-Service | where-object {$_.DisplayName -like "*sysm*"}
+      Get-CimInstance win32_service -Filter "Description = 'System Monitor service'"
+    name: powershell
```

Comment:
I would prefer to leave this one as is and instead move your additions to the first test where you added several other commands like "sc query windefend" for example

Reply:
@clr2of8, I have made necessary changes. Thank you